sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -187,6 +192,15 @@ export class ApplicationStudentsController extends BaseController {
       const offering = await this.offeringService.getOfferingById(
         payload.data.selectedOffering,
       );
+      if (
+        !isFulltimeAllowed &&
```

Comment:
The validation is happening only when an offering is selected in the application. It must happen otherwise as well.

Reply:
Isn't it that when the offering is not selected in the application, the `formService.dryRunSubmission` will fail resulting in the `BadRequestException`. Please let me know if I missed something.

Reply:
@sh16011993 what @dheepak-aot is saying is what we talked about in the call between the 3 of us.
Either the dryRun must ensure that only Part-time intensity would be selected (which) or we need to add a code here to check it. I believe that the decision was to add the code check.

Reply:
Changing it as per the discussion.